addons/point_of_sale/views/pos_ticket_view.xml

```diff
@@ -92,7 +92,7 @@
                                     <h4>Please enter your billing information <small class="text-muted">or</small> <a role="button" t-att-href="'/web/login?redirect=/pos/ticket/validate?access_token=%s' % access_token" style="margin-top: -11px"> Sign in</a>:</h4>
                                 </div>
                             </div>
-                            <div class="row o_portal_details">
+                            <div class="row o_customer_address_fill">
```

Comment:
Hey 

This isn’t stable; users can’t install `l10n_{ar/pe/ec_edi}_pos` if they’ve already installed `point_of_sale` before this commit (`o_customer_address_fill` is therefore not present in the parent view).

Can you do something asap ? Because we have a lot of impacted db :/

CC @adwid

Reply:
Hello @adwid 

A PR has already been created for this by @aamo-odoo 
https://github.com/odoo/enterprise/pull/87729
https://github.com/odoo/odoo/pull/214339